osu.Game/Rulesets/Objects/Legacy/ConvertHitObjectParser.cs

```diff
@@ -335,11 +335,14 @@ private PathControlPoint[] convertPathString(string pointString, Vector2 offset)
                 ArrayPool<(PathType, int)>.Shared.Return(segmentsBuffer);
             }
 
-            static Vector2 readPoint(string value, Vector2 startPos)
+            Vector2 readPoint(string value, Vector2 startPos)
             {
                 string[] vertexSplit = value.Split(':');
 
-                Vector2 pos = new Vector2((int)Parsing.ParseDouble(vertexSplit[0], Parsing.MAX_COORDINATE_VALUE), (int)Parsing.ParseDouble(vertexSplit[1], Parsing.MAX_COORDINATE_VALUE)) - startPos;
```

Comment:
I'm reasonably confident that `ParseDouble()` was gratuitous here because `IHas[XY]?Position` are all single precision-based anyway.